### DISCUSSION TITLE
The new 2FA requirement is foolish

### DISCUSSION THREAD

**Discussion by WingsLikeEagles** [community/community#84892] [Other Feature Feedback, Questions, & Ideas]
2FA has been shown over and over to be flawed in preventing hacking and account abuse.  Please stop being stupid and just blindly following trends.  Requiring 2FA will NOT enhance security.  It just increases the ability to track who logs in and their location.  This is an invasion of privacy.  If GitHub continues down this road I will shut down my account.

**Comment by ghostinhershell** [comment]
Hey there! 👋🏾 We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.
